lib/repl.js

```diff
@@ -361,7 +361,7 @@ function REPLServer(prompt,
 
     // Check to see if a REPL keyword was used. If it returns true,
     // display next prompt and return.
-    if (cmd && cmd.charAt(0) === '.' && isNaN(parseFloat(cmd))) {
+    if (cmd && isNaN(parseFloat(cmd)) && /^\.[a-z]+\s*[^(]*$/.test(cmd)) {
       var matches = cmd.match(/^\.([^\s]+)\s*(.*)$/);
```

Comment:
> Also `_` and `-` are not necessary in it, because the REPL commands don't have them.

The defaults don't currently, correct. But they *could* eventually. Also I think the repl can be extended to add more commands? If so, those characters seem like likely candidates to exist in a custom command. While we are at it we should add `A-Z` to the match too.

Given the above, it's possible this would technically be a breaking change then? I'm not so sure, or concerned either.